source/gui/settingsDialogs.py

```diff
@@ -1990,7 +1991,8 @@ def __init__(self, parent):
 			parent=self,
 			sizer=wx.StaticBoxSizer(parent=self, label=groupText, orient=wx.VERTICAL)
 		)
-		sHelper.addItem(devGroup)
+		if not config.isAppX:
+			sHelper.addItem(devGroup)

```

Comment:
Surely we should keep the grouping and controls around, but just disable the controls, like you are already doing below?